ironmq/src/main/resources/reference.conf

```diff
@@ -0,0 +1,33 @@
+akka.stream.alpakka.ironmq {
+
+  // The IronMg endpoint. It may vary due to availability zone and region.
+  endpoint = "https://mq-aws-eu-west-1-1.iron.io"
+
+  credentials {
+
+    // The IronMq project id
+    // project-id =
+
+    // The IronMq auth token
+    // token =
+  }
+
+  consumer {
+
+    // This is the max number of message to fetch from IronMq.
+    buffer-max-size = 100
+
+    // This is the threshold where fech other messages from IronMq
+    buffer-min-size = 25
+
+    // This is the time interval between each poll loop
+    fetch-interval = 250 milliseconds
+
+    // This is the amount of time the IronMq client will wait for a message to be available in the queue
+    poll-timeout = 0
+
+    // This is the amount of time a fetched message will be not available to other consumers
+    reservation-timeout = 30 seconds
+
+  }
+}
```

Comment:
a newline character is actually important in those conf files, because otherwise it will be wrong when using tools like sbt-assembly that merges those files